app-modules/alert/database/factories/AlertFactory.php

```diff
@@ -51,17 +50,7 @@ class AlertFactory extends Factory
     public function definition(): array
     {
         return [
-            'concern_type' => $this->faker->randomElement([(new Contact())->getMorphClass()]),
-            'concern_id' => function (array $attributes) {
-                $concernClass = Relation::getMorphedModel($attributes['concern_type']);
-
-                /** @var Contact $concernModel */
-                $concernModel = new $concernClass();
-
-                $concern = $concernModel::factory()->create();
-
-                return $concern->getKey();
-            },
+            'concern_id' => Contact::factory()->create()->getKey(),
```

Comment:
Let's change this to just be the factory so that Laravel automatically populates it.

```suggestion
            'concern_id' => Contact::factory(),
```